app/javascript/utils/form/final_form/Field.js

```diff
@@ -133,7 +132,18 @@ export const CheckboxField = ({ fieldName, label, blockNote, validation, id, ari
   </Field>
 )
 
-export const YesNoRadioField = ({ fieldName, uniqId, trueValue = 'true', trueLabel = 'Yes', falseValue = 'false', falseLabel = 'No', inputClassName, className, label }) => {
+const YesNoRadioField = ({ value, label, type, fieldName, className, uniqId }) => (
```

Comment:
curious where these updates to YesNoRadio field/group came from in this PR. if just a refactor or potential regression

Reply:
These were due to code climate failing on similar code issues, refactored to meet the standard